packages/components/package.json

```diff
@@ -35,7 +35,7 @@
     "classnames": "^2.2.6",
     "color": "^3.1.2",
     "filesize": "^6.1.0",
-    "framer-motion": "^1.10.3",
+    "framer-motion": "6.3.16",
```

Comment:
Locked to this version because 6.4 and above use another 3rd party package that requires the latest version of typescript. Which we don't have and the compilation will fail